packages/zoe/src/zoeService/offer/offer.js

```diff
@@ -34,15 +35,25 @@ export const makeOfferMethod = (
     paymentKeywordRecord = harden({}),
     offerArgs = undefined,
   ) => {
-    const { instanceHandle, invitationHandle } = await burnInvitation(
-      invitationIssuer,
+    const query = makeInvitationQueryFns(invitationIssuer);
+    const { instance, description } = await query.getInvitationDetails(
       invitation,
     );
     // AWAIT ///
 
-    const instanceAdmin = getInstanceAdmin(instanceHandle);
+    const instanceAdmin = getInstanceAdmin(instance);
+    assert(
+      !instanceAdmin.isSnoozed(description),
+      `not accepting offer with description '${description}'`,
```

Comment:
this reads like the particular offer is being rejected and another might be accepted.

shouldn't it use details/quote?
```suggestion
      X`contract is not currently accepting offers. ${q(description)}`,
```

Reply:
ah, I misread `updateSnoozedList`. That method name could be improved.

For one, isn't it setting the list? There's no history or partial update; it's a full replacement.

For two, I maintain that "snooze" is too metaphorical for the concept of "stop accepting offers whose descriptions contain one of these strings". (Incidentally, that could make a good JSdoc for the method, if I got it right)

Three, "list" is vague. List of what? Not snoozes.  Consider "selectors". Or since it's about what to reject, "filter"

Putting it all together, maybe `setOfferFilter`. Where the filter is a list of substrings that can match.

Reply:
done